lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -560,6 +569,55 @@ def upload_csv_to_report_store(rows, csv_name, course_id, timestamp, config_name
     tracker.emit(REPORT_REQUESTED_EVENT_NAME, {"report_type": csv_name, })
 
 
+def upload_exec_summary_to_store(data_dict, report_name, course_id, generated_at, config_name='FINANCIAL_REPORTS'):
+    """
+    Upload Executive Summary Html file using ReportStore.
+
+    Arguments:
+        data_dict: containing executive report data.
+        report_name: Name of the resulting Html File.
+        course_id: ID of the course
+    """
+    report_store = ReportStore.from_config(config_name)
+
+    # Embed the brand and the cobrand logos in the generated html in base64 encoding.
+    data_dict['cobrand_logo_data'] = None
+    if data_dict['cobrand_logo_path']:
+        try:
+            logo_data_binary = open(data_dict['cobrand_logo_path'], 'rt').read()
+            encoded_data = base64.b64encode(logo_data_binary)
+            data_dict['cobrand_logo_data'] = encoded_data
+        except IOError, ex:
+            TASK_LOG.exception(ex)
+
+    data_dict['logo_data'] = None
+    if data_dict['logo_path']:
+        try:
+            logo_data_binary = open(data_dict['logo_path'], 'rt').read()
+            encoded_data = base64.b64encode(logo_data_binary)
+            data_dict['logo_data'] = encoded_data
+        except IOError, ex:
+            TASK_LOG.exception(ex)
```

Comment:
@afzaledx @muhhshoaib I added some code here (and in template itself) to be more resilient if there's no specified logo information (e.g. someone running a simple Open edX instance). Hopefully this should get the builds back working. That said, I'm concerned about two things:

1) This approach of base64 encoding the image into the HTML could be problematic for images that are large.
2) The HTML template asserts that the image needs to be PNG, which may not be the case

Honestly, I'm not sure we should keep this. If we do, then we should check (and limit) the allowed size, and we should compute the mime-type (e.g. image/png) and pass it to the template.

That said, I think this is a bunch of work and might be best to simply remove these logos for now so we can wrap this up.

Thanks